src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.services.Color;
+
+public class IsoscelesTrapezoid extends Figure {
+    public int firstSide;
+    public int secondSide;
+    public int height;
+
+    public IsoscelesTrapezoid(Color color, int firstSide, int secondSide, int height) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+    }
+
+    @Override
+    public float getArea() {
+        return (float) (this.firstSide + this.secondSide) / 2 * this.height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + this.getArea() + "sq. units, firstSide: "
+                + this.firstSide + " units, secondSide: " + this.secondSide + " units, height: "
```

Comment:
You can use attributes without ```this``` keyword inside the methods in the same class :)
Please remove it here and in other places

Reply:
Thanks for your advice, done!